Ruleset/research_XCOMFILES.rul

```diff
@@ -5998,13 +5996,14 @@
     destroyItem: true
     unlocks:
       - STR_ALIEN_POWER_SYSTEMS_DEP_1
-      - STR_ALIEN_MEDICS
+      - STR_ALIEN_MEDIC_INTERROGATED
```

Comment:
Also, delet this!
Ugh!

Reply:
I still have no idea what modularization is and why it would be good. Please give me a real answer or stop wasting my time. I am trying to get engaged, I appreciate your effort, but please also respect mine.

From the article excerpt:
> This makes modular designed systems, if built correctly, far more reusable than a traditional monolithic design
I have no idea what it means.
> This also facilitates the "breaking down" of projects into several smaller projects.
I am not interested in that, XCF is one project, managed by one person. Any extra content goes through me anyway.
> And because problems can be pin pointed to fewer files, what makes managing changes less stressful.
I am completely unstressed as I am now.
> You can also remove or refactor entire segments of code without forgetting anything, because it is contained within a set of folders/files.
No idea what you're referring to.
> It allows for parts of the mod to be isolated for future use on other mods without stress.
No, it will be mess no other modder will want to touch.
Look, I know many mothers, some of them capable programmers, and nobody uses these methods.

Also:
STR_ALIEN_MEDICS - why would it be deleted?

Reply:
> Also:
> STR_ALIEN_MEDICS - why would it be deleted?

Because it is under muton medic, which are muton guards.
I am just saying to also delete my addition.

Reply:
Ah, I see now!